app/adapters/signal_adapter/api.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module SignalAdapter
+  class Api
+    class << self
```

Comment:
I guess a class with a static method will do. `ErrorNotifier` is built the same way`.

Reply:
Sure, don't know why I didn't think of that hehe